index.html

```diff
@@ -1,9 +1,134 @@
+  }
+  .step strong {
+    font-weight: 500;
+  }
+  .step:hover {
+    cursor: pointer;
+    background: #eee;
+  }
+  </style>
+
   </head>
+
+
   <body>
-    YOUR PROJECT HERE
+
+    <div class="container" ng-controller="Main" ng-init="init()">
+
+      <h1 align="center">How To Ride The Subway</h1>
+      <h2 align="center">As you advance, click for the next step in the guide.</h2>
+
+      <div class="row">
+        <div class="step" style="margin-top: 20px;" ng-click="underground = 'true'"><strong>Step One:</strong> Go Underground</div>
+      </div>
+
+      <div ng-show="underground">
+
+        <div class="row">
+          <div class="step"><strong>Step Two:</strong> Follow The Signs Leading To Your Train</div>
+        </div>
+
+        <div class="row">
+          <div class="col-1-2"><div class="step red" ng-click="lost = 'true'">I'm Lost</div></div>
+          <div class="col-1-2"><div class="step green" ng-click="lost = 'false'">Got It!</div></div>
+        </div>
+
+      </div>
+
+      <div class="row" ng-show="lost == 'true'">
+        <div class="col-1-1">
+          <div class="step"><strong>You're Lost?</strong><br />Go back and carefully follow the signs pointing to your train.</div>
+        </div>
+        <div class="col-1-1">
+          <div class="step green" ng-click="lost = 'false'">Got It!</div>
+        </div>
+      </div>
+
+      <div class="row" ng-show="lost == 'false'" ng-click="board = 'true'"><div class="step"><strong>Step Three:</strong> Wait On The Platform For The Next Train</div></div>
+
+      <div class="row" ng-show="board" ng-click="ride = 'true'"><div class="step"><strong>Step Four:</strong> When the train pulls up, be careful not to bump others as you board.</div></div>
+
+      <div ng-show="ride">
+
+        <div class="row">
+          <div class="step"><strong>Step Five:</strong> Watch for your stop! On newer trains, your stop will light up on the boards near the top of the train.</div>
+        </div>
+
+        <div class="row">
+          <div class="col-1-2"><div class="step red" ng-click="missedStop = 'true'">I Missed My Stop</div></div>
+          <div class="col-1-2"><div class="step green" ng-click="missedStop = 'false'">Got It!</div></div>
+        </div>
+
+      </div>
+
+      <div class="row" ng-show="missedStop == 'true'">
+        <div class="col-1-1">
+          <div class="step"><strong>Missed Your Stop?</strong><br />Don't Panic! Look at the map on the subway and find a place that you can transfer to another train. Only turn around if there are no other options.</div>
+        </div>
+        <div class="col-1-1">
+          <div class="step green" ng-click="missedStop = 'false'">Got It!</div>
+        </div>
+      </div>
+
+      <div class="row" ng-show="missedStop == 'false'"><div class="step"><strong>Step Six:</strong> Get off at your stop and enjoy your day!</div></div>
+
+      <div class="row" align="center"><a href="#" id="reset">Reset</a></div>
+
+    </div>
```

Comment:
This DOM setup is fairly repetitive... how would you make it more [DRY](http://en.wikipedia.org/wiki/Don%27t_repeat_yourself)?